tests/test_ec2/test_key_pairs.py

```diff
@@ -101,6 +104,9 @@ def test_key_pairs_create_boto3():
     kps[0].should.have.key("KeyPairId")
     kps[0].should.have.key("KeyName").equal(key_name)
     kps[0].should.have.key("KeyFingerprint")
+    kps[0]["CreateTime"].replace(tzinfo=None).should.equal(
+        datetime(2014, 1, 1, 5, 0, 0)
+    )
```

Comment:
Probably best to just follow the existing convention (`should.have.key`) and not try to assert against the exact date/time (see my previous comment about using `freeze_time`).  The important thing to check is that the date/time moto passes back is properly deserialized by boto into a `datetime` object, e.g. something like `kps[0].should.have.key("CreateTime").should.be.a(datetime)`